docs/design/keyboard-shortcuts.md

```diff
@@ -281,31 +279,31 @@ Office.actions.replaceShortcuts(userCustomShortcuts)
+
+The following is an example.
 
 ```javascript
 Office.actions.getShortcuts()
     .then(function (userShortcuts) {
        for (const action in userShortcuts) {
            let shortcut = userShortcuts[action];
-           // In a real add-in you would typically present this information in the UI.
            console.log(action + ": " + shortcut);
        }
     });
 
 ```
 
-As described in [Avoid key combinations in use by other add-ins](#avoid-key-combinations-in-use-by-other-add-ins), the first time a user enters a key combination that is registered to multiple actions, they are prompted to choose which action will be associated with the combination. This can be a dilemma for a user that wants more than one of the actions to have a shortcut. You can't prevent this from happening when the conflict is between your add-in and another add-in, but you can at least prevent conflicts between the *user's customization of one of your add-in's shortcuts* and a shortcut registered by an add-in or the host Office application. To prevent a conflict, you need to warn the user about it and give them the chance to change their custom key combination.
-
-To do this, pass the user's custom key combinations as an array of strings to the [Office.actions.areShortcutsInUse](/javascript/api/office/office.actions#areShortcutsInUse) method. The method returns a report containing key combinations that are already in use in the form of an array of objects of type `{shortcut: string, inUse: boolean}`. The `shortcut` property is a key combination, such as "CTRL+SHIFT+1". If the combination is already registered to another action, the `inUse` property is set to `true`. For example, `[{shortcut: "CTRL+SHIFT+1", inUse: true}, {shortcut: "CTRL+SHIFT+2", inUse: false}]`. The following code snippet is an example in which some hypothetical user's custom key combinations are hardcoded. In an actual add-in, you programmatically get this information from the UI where the user has entered custom key combinations. Also, instead of just logging the results to the console, you would typically report the results to the UI and give the user a way to change key combinations that are already in use.
+To discover if one or more key combinations are already in use pass them as an array of strings to the [Office.actions.areShortcutsInUse](/javascript/api/office/office.actions#areShortcutsInUse) method. The method returns a report containing key combinations that are already in use in the form of an array of objects of type `{shortcut: string, inUse: boolean}`. The `shortcut` property is a key combination, such as "CTRL+SHIFT+1". If the combination is already registered to another action, the `inUse` property is set to `true`. For example, `[{shortcut: "CTRL+SHIFT+1", inUse: true}, {shortcut: "CTRL+SHIFT+2", inUse: false}]`. The following code snippet is an example:
```

Comment:
Would be great to somehow link this to the "Avoid key combinations..." section. Maybe just a sentence about using this API to avoid conflicts with built in Excel shortcuts?